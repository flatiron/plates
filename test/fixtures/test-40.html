<ul>
  <li class="zero"></li>
  <li class="one"></li>
  <li class="banana"></li>
</ul>
