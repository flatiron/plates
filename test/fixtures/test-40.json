{
  "counts": {
    "zero": 0,
    "one": 1
  },
  "ringring": "bananaphone"
}
