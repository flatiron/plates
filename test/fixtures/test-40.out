<ul>
  <li class="zero">0</li>
  <li class="one">1</li>
  <li class="banana">bananaphone</li>
</ul>
